cloud-gateway-service/src/test/java/org/zowe/apiml/cloudgatewayservice/acceptance/common/AcceptanceTestWithBasePath.java

```diff
@@ -23,6 +24,7 @@ public class AcceptanceTestWithBasePath {
     @BeforeEach
     void setBasePath() {
         basePath = String.format("https://localhost:%d", port);
+        RestAssured.useRelaxedHTTPSValidation();
```

Comment:
Weren't these tests using "properly" configured certificates? If it's relaxed now, it may be skipping some checks

Reply:
Yes, but for v2 I would prefer just skip that. It is only checking against the truststore. The trustore is not set in there. If you think it is critical I can update the calls.